src/Script/Composer/Composer.php

```diff
@@ -6,11 +6,23 @@
 
 class Composer
 {
-    public static function removeUnusedServices(
-        Event      $event,
-        Filesystem $filesystem = null
-    )
+
+    public static function removeUnusedServicesInDevOnly(Event $event, Filesystem $filesystem = null)
     {
+        self::removeUnusedServicesByExtraConfig($event, $filesystem, true);
+    }
+
+    public static function removeUnusedServices(Event $event, Filesystem $filesystem = null)
+    {
+        self::removeUnusedServicesByExtraConfig($event, $filesystem, false);
+    }
+
+    private static function removeUnusedServicesByExtraConfig(Event $event, Filesystem $filesystem = null, $onlyRunInDev = false)
```

Comment:
Would you please rename the method from `removeUnusedServicesByExtraConfig` to `removeUnusedServicesWithConfig`.

Would you please also rename the parameter `$onlyRunInDev` to `$isDev`.